tools/paconn-cli/paconn/operations/download.py

```diff
@@ -47,9 +47,8 @@ def _prepare_directory(destination, connector_id):
     # Create a sub-directory in the current directory
     # when a destination isn't provided
     else:
-        if os.path.isdir(connector_id):
-            error = '{} directory already exists. Please remove the directory before continuing.'
-            raise CLIError(error.format(connector_id))
+        if os.path.isdir(connector_id):            
```

Comment:
Please simplify the logic a bit more like following:

```python
        if not os.path.isdir(connector_id):
            os.mkdir(connector_id)
        destination = connector_id
```

Reply:
Thanks for the feedback!